src/usr/share/gpgmailer/gpgmailer-daemon.py

```diff
@@ -323,8 +322,15 @@ def send_expiration_warning_message(gpg_keyring, config, expiration_date):
         mail_message = gpgmailmessage.GpgMailMessage()
         mail_message.set_subject(config['default_subject'])
         mail_message.set_body(message)
+        recipient_emails = []
+        recipient_keys = []
+        for recipient in config['recipients']:
+            recipient_emails.append(recipient['email'])
+            recipient_keys.append(recipient['fingerprint'])
```

Comment:
This should be the e-mail addresses and fingerprints from the keyring. (Or maybe just the e-mail addresses depending on what we decide.)